parm/atm/obs/config/sondes.yaml

```diff
@@ -1,74 +1,44 @@
 obs space:
   name: sondes
   obsdatain:
-    engine:
+    engine: 
       type: H5File
-      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sondes.$(OBS_DATE).nc4
-    obsgrouping:
-      group variables: ["station_id"]
-      sort variable: "air_pressure"
-      sort order: "descending"
+      obsfile: sondes_obs_$(OBS_DATE).nc4
   obsdataout:
     engine:
       type: H5File
-      obsfile: $(DIAG_DIR)/diag_sondes_$(OBS_DATE).nc4
-  io pool:
-    max pool size: 1
-  simulated variables: [eastward_wind, northward_wind, air_temperature, specific_humidity, surface_pressure]
-get values:
-  interpolation method: $(INTERP_METHOD)
+      obsfile: sondes_diag_$(OBS_DATE).nc4
+  simulated variables: [surface_pressure, air_temperature, eastward_wind, northward_wind, specific_humidity]
+geovals:
+  filename: sondes_geoval_$(OBS_DATE).nc4
+vector ref: GsiHofXBc
+tolerance: 0.01
 obs operator:
   name: Composite
   components:
    - name: VertInterp
      variables:
      - name: air_temperature
-#    - name: virtual_temperature
-     - name: specific_humidity
      - name: eastward_wind
      - name: northward_wind
+     - name: specific_humidity
    - name: SfcPCorrected
-     variables:
-     - name: surface_pressure
      da_psfc_scheme: GSI
-linear obs operator:
-  name: Composite
-  components:
-   - name: VertInterp
-     variables:
-     - name: air_temperature
-     - name: specific_humidity
-     - name: eastward_wind
-     - name: northward_wind
-   - name: Identity
-     variables:
+     geovar_sfc_geomz: surface_geometric_height
+     geovar_geomz: geopotential_height
+     variables: 
      - name: surface_pressure
-#
+linear obs operator:
+  name: Identity
 obs filters:
   #
   # Reject all obs with PreQC mark already set above 3
-  # - filter: PreQC
-  #   maxvalue: 3
-  #   action:
-  #     name: reject
-  #
-  # Observation Range Sanity Check
-  - filter: Bounds Check
-    filter variables:
-    - name: air_temperature
-    minvalue: 195
-    maxvalue: 327
+  - filter: PreQC
```

Comment:
Is there a reason why there is like almost no QC retained in this file?

Reply:
I mostly removed all the QC for all the variables, except surface_pressure, and applied the PreQC filter. It worked better compared to the old YAML. For a comparison of the number of obs passing QCs with different YAMLs (no QC, old YAML, and new YAML), please look at the following document: https://docs.google.com/document/d/1_w6P-cJkxyYzTvLNMm6kCBWj2ZATk0PMsXOdKW29z8s/edit.

Reply:
Please look at my sonde obs yaml PR.